fence/auth.py

```diff
@@ -1,23 +1,27 @@
-import flask
-from flask_sqlalchemy_session import current_session
+import urllib.error
+import urllib.parse
+import urllib.request
```

Comment:
We need to keep Fence in sync with the master, it is easier adding things than editing/removing part. Please reduce changes to imports that are not necessary.

Reply:
These are still being imported, nothing removed.